test/components/labelTests.ts

```diff
@@ -2,7 +2,7 @@
 
 describe("Labels", () => {
   const LABEL_CLASS = "label";
-  let CLOSETO_REQUIRMENT;
+  let CLOSETO_REQUIRMENT: number;
```

Comment:
Not sure. I cherry-picked the commit here before the other PR merged, so it's possible Github got confused.